integration/test/Test/FeatureFlags/User.hs

```diff
@@ -0,0 +1,70 @@
+module Test.FeatureFlags.User where
+
+import qualified API.BrigInternal as I
+import API.Galley
+import qualified API.GalleyInternal as I
+import SetupHelpers
+import Test.FeatureFlags.Util
+import Testlib.Prelude
+
+testFeatureConferenceCallingForUser :: App ()
+testFeatureConferenceCallingForUser = do
+  (alice, tid, _) <- createTeam OwnDomain 0 -- team user
+  bob <- randomUser OwnDomain def -- non-team user
+  let featureName = "conferenceCalling"
+      ttl = Just (toJSON (360000 :: Int))
+
+  -- set initial value at the team level
+  let initial =
+        confCalling
+          def
+            { status = "enabled",
+              ttl = ttl,
+              sft = toJSON True
+            }
+  assertSuccess =<< I.setTeamFeatureConfig OwnDomain tid featureName initial
+
+  -- set user value for both users
+  for_ [alice, bob] $ \u -> do
+    void $ I.putFeatureForUser u featureName (object ["status" .= "disabled"]) >>= getBody 200
+    config <- I.getFeatureForUser u featureName >>= getJSON 200
+    config %. "status" `shouldMatch` "disabled"
+    config %. "ttl" `shouldMatch` "unlimited"
+
+    -- this config is just made up by brig, it does not reflect the actual value
+    -- that will be returned to the user
```

Comment:
i don't understand.  are you saying the internal end-point returns a different config from the public end-point for the same user?  isn't that a bad thing?

Reply:
It is a bad thing, but it was already there. The problem is that brig returns a whole `WithStatus` object, despite only having information about the status flag. We can try to improve this with the coming refactoring.